db/models/applications.js

```diff
@@ -30,6 +31,16 @@ class PendingApplication extends Model {
                 allowNull: false
             },
 
+            /**
+             * Application's Current Progress Status
+             * e.g. PENDING
+             */
+            currentProgressState: {
+                type: DataTypes.ENUM('NOT_STARTED', 'PENDING', 'COMPLETE'),
+                allowNull: true,
+                defaultValue: 'NOT_STARTED'
```

Comment:
Need to check if this does not replace the existing in progress applications to have `NOT_STARTED` instead of leaving it as `NULL`.

Reply:
I think it will only take effect if we created a new application and didn't specify this field – when we add the new column manually, it'll set everything already in there to `NULL`.

Reply:
Thats what I assumed but running the query `... MODIFY currentProgressState enum('NOT_STARTED', 'PENDING', 'COMPLETE') DEFAULT 'NOT_STARTED'` changed all the existing applications to `NOT_STARTED` too. My only guess is MYSQL `DEFAULT` is not the same as Sequelize `defaultValue` ?

Reply:
Ah okay, maybe when we manually create the column we'll have to set it to `DEFAULT NULL` then immediately change the default (thoughts @davidrapson)?

Reply:
I think default of `null` is OK, and we could have the new applications method explicitly set NOT_STARTED?